### DISCUSSION TITLE
Redis changed licenses

### DISCUSSION THREAD

**Discussion by melroy89** [dragonflydb/dragonfly#2757] [General]
Redis now adopt dual licensing: https://redis.com/blog/redis-adopts-dual-source-available-licensing/

Basically another very good reason to go away from Redis, so expect more people now to look for alternatives like Dragonfly!

I do want to mention that some users might want to install Dragonfly on "bare metal" without Docker. So please, extend your docs to support those kind of installations as well [in your quick-start guide](https://github.com/dragonflydb/dragonfly/tree/main/docs/quick-start). Maybe point them to: https://www.dragonflydb.io/docs/getting-started/binary ?

**Comment by melroy89** [comment]
Maybe you might want to extend your documentation regarding DEB installation as well.

**Reply by romange** [reply]
what do you mean? here? https://www.dragonflydb.io/docs/getting-started

**Reply by melroy89** [reply]
No here: https://github.com/dragonflydb/dragonfly/tree/main/docs/quick-start (this is what people find if they click on quick-start from [your repo](https://github.com/dragonflydb/dragonfly) under "Contents" header)

**Reply by melroy89** [reply]
[This quick start document](https://github.com/dragonflydb/dragonfly/tree/main/docs/quick-start) is only behind the link under "Contents". And that specific quick-start document hasn't been update for 2 years.

NOTE: The link with the **same name** but at the top of the repo does go to the https://www.dragonflydb.io site...

**Reply by romange** [reply]
We will need to fix it indeed. Will probably just delete the file in the repo and align the link to our docs site. Thanks for pointing out!
If you want to fix it you are more than welcome to send a PR otherwise will handle it on Sunday.
